api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java

```diff
@@ -1980,6 +2035,43 @@ public boolean hasAnyConceptAttribute(ConceptAttributeType conceptAttributeType)
 		return dao.getConceptAttributeCount(conceptAttributeType) > 0;
 	}
 
+	/***
+	 * Determines if the passed string is in valid uuid format By OpenMRS standards, a uuid must be 36
+	 * characters in length and not contain whitespace, but we do not enforce that a uuid be in the
+	 * "canonical" form, with alphanumerics seperated by dashes, since the MVP dictionary does not use
+	 * this format (We also are being slightly lenient and accepting uuids that are 37 or 38 characters
+	 * in length, since the uuid data field is 38 characters long)
+	 */
+	public static boolean isValidUuidFormat(String uuid) {
+		if (uuid.length() < 36 || uuid.length() > 38 || uuid.contains(" ") || uuid.contains(".")) {
+			return false;
+		}
+
+		return true;
+	}
+
+	/**
+	 * Evaluates the specified Java constant using reflection
+	 * 
+	 * @param fqn the fully qualified name of the constant
+	 * @return the constant value
+	 */
+	private static String evaluateStaticConstant(String fqn) {
```

Comment:
What kind of evaluation is this? In your javadoc, can you give some examples of input and output values?

Reply:
This is pretty much taken as-is from the HTMLFormEntry version. In particular, this is meant to deal with [HTML-524](https://issues.openmrs.org/browse/HTML-524).

Reply:
@ibacher @dkayiwa Should I eliminate the functionality of getting concept using static constant.